### PR TITLE
Translationfr

### LINKED ISSUE #13: Import FR catalog data

### PR DESCRIPTION
**Pull request by AxioBen**
Fixes #13

### CODE REVIEW

File: app/views/event.sets.edit.html [hunk 1]
```diff
@@ -88,7 +88,7 @@ <h5>Add item from catalogue</h5>
           <td>{{item.item.model}}</td>
           <td><input class='form-control input-sm' type='number' ng-model="item.quantity" /></td>
           <td><select class="form-control input-sm" ng-model="item.multiplier" ng-options="multi.id as multi.name for multi in multipliers"></select></td>
-          <td><input name='multiply_factor' type="number" class="input-sm form-control" placeholder="X number" ng-model='item.multiply_factor' required="Multiply factor required" ng-required="true"></td>
+          <!-- <td><input name='multiply_factor' type="number" class="input-sm form-control" placeholder="X number" ng-model='item.multiply_factor' required="Multiply factor required" ng-required="true"></td> -->
```
Comment (fabian) [inline app/views/event.sets.edit.html]: Correct me if I'm wrong, but I think this should not be commented out?

File: app/languages/en_US.json [hunk 2]
```diff
@@ -0,0 +1,400 @@
+{
+  "JSTEXT": {
```
[the remark this answers is not in the document]
Comment (AxioBen) [reply app/languages/en_US.json]: We use it with our integrator/translator to identify each keys needed to add in js files (with $translator). But now it's outdated i can remove this node.

File: app/scripts/config.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
+(function() {
```
[the remark this answers is not in the document]
Comment (AxioBen) [reply app/scripts/config.js]: Don't know why it's here, i will remove it.

File: app/views/event.sets.edit.html (hunk 1 above)
Comment (AxioBen) [reply app/views/event.sets.edit.html]: No, you are right, it's a mistake.

**Pull request merged**